openwisp_radius/apps.py

```diff
@@ -104,13 +105,61 @@ def radiusorgsettings_post_save(self, instance, **kwargs):
     def radiusorgsettings_post_delete(self, instance, **kwargs):
         instance.delete_cache()
 
-    def add_default_menu_items(self):
-        menu_setting = 'OPENWISP_DEFAULT_ADMIN_MENU_ITEMS'
-        items = [
-            {'model': f'{self.label}.RadiusAccounting', 'label': _('Radius sessions')}
-        ]
-        if not hasattr(settings, menu_setting):
-            setattr(settings, menu_setting, items)
-        else:
-            current_menu = getattr(settings, menu_setting)
-            current_menu += items
+    def regiser_menu_groups(self):
+        register_menu_group(
+            position=30,
```

Comment:
let's make sure the radius module comes after organization and users.

Reply:
Okay, I will change it as now we are registering org and users at 40 now